common/logging/log4j2/pom.xml

```diff
@@ -24,9 +24,5 @@
           <groupId>org.apache.logging.log4j</groupId>
           <artifactId>log4j-api</artifactId>
         </dependency>
-        <dependency>
-          <groupId>org.apache.logging.log4j</groupId>
-          <artifactId>log4j-core</artifactId>
```

Comment:
@msavy I think we're just waiting on confirmation from you on this one.  When you get a chance. :)